R/corrplot.R

```diff
@@ -297,6 +304,26 @@ corrplot <- function(corr,
     addgrid.col <- switch(method, color = NA, shade = NA, "grey")
   }
 
+
+  if (mreg) {
+    # mreg code: number of response variables usually smaller than number of
+    # predictor variables
+    is.corr <- FALSE
+    #if (is.null(cl.lim)) {
+      # if the matrix is expected to be a multiple regression correlation matrix
+      # it MUST be within the interval [-1,1], although it can be overruled
+    #  cl.lim <- c(-1,1)
+    #}
+  }
+
+  # mreg code: catch corr for multiple regression
+  # cross out of ignored predictors
+  corrbefore <- corr
```

Comment:
looks like this should be inside the if statement.

Reply:
Good idea, saves memory when mreg=FALSE